test/unit_Translation.jl

```diff
@@ -0,0 +1,51 @@
+for N in [Float64, Rational{Int}, Float32]
+
+    # ==================================
+    # Constructor and interface methods
+    # ==================================
+ 
+    B = BallInf(N[2, 2, 2], N(1))
+    v = N[0, 0, 0]
```

Comment:
Please use some other vector to actually test the correct behavior.

Reply:
Good. I added a translation along dimension 1 of a 3d set.